styleguide/gulpfile.js

```diff
@@ -132,6 +132,14 @@ gulp.task('icons', function (callback) {
   runSequence('minifyIcons', 'makeIcons', 'waitForIcons', 'reloadIcons', callback);
 });
 
+//Task: Copy css to public
+gulp.task('css',function() {
```

Comment:
This task is looking for CSS files in our assets directory. But reset is an scss file.